cosmic-metrics-collector/src/main/java/com/github/missioncriticalcloud/cosmic/metricscollector/repositories/jdbc/PublicIPAddressMetricsJdbcRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.github.missioncriticalcloud.cosmic.metricscollector.repositories.jdbc;
+
+import com.github.missioncriticalcloud.cosmic.metricscollector.model.Metric;
+import com.github.missioncriticalcloud.cosmic.metricscollector.repositories.PublicIPAddressMetricsRepository;
+import com.github.missioncriticalcloud.cosmic.metricscollector.repositories.mappers.PublicIPAddressMetricsMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Properties;
+
+@Repository
+public class PublicIPAddressMetricsJdbcRepository implements PublicIPAddressMetricsRepository {
```

Comment:
I suggest to use the _spring-like_ camel-case for names wherever it applies. For example: `PublicIpAddressMetricsJdbcRepository`.